lib/src/source/hosted.dart

```diff
@@ -537,6 +537,62 @@ class HostedSource extends CachedSource {
     return result;
   }
 
+  Future<_PackageAdvisories> _fetchAdvisories(
+    _RefAndCache refAndCache,
```

Comment:
Don't use the _RefAndCache type, it is only used because records was not a thing back then - just take a Ref and a SystemCache.